manim/animation/numbers.py

```diff
@@ -34,10 +69,39 @@ def check_validity_of_input(self, decimal_mob: DecimalNumber) -> None:
             raise TypeError("ChangingDecimal can only take in a DecimalNumber")
 
     def interpolate_mobject(self, alpha: float) -> None:
+        """Interpolates the mobject of the Animation based on alpha value.
+
+        Parameters
+        ----------
+        alpha
+            A float between 0 and 1 representing the animation progress.
+        """
```

Comment:
It should not be necessary to add a docstring to this method, since the overriden `Animation.interpolate_mobject()` already has a docstring.
```suggestion
```